cves/CVE-2013-0863.yml

```diff
@@ -342,7 +352,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: Lack of testing. Unit testing should have checked for out of bounds exceptions.
```

Comment:
This makes sense but feels underdeveloped. Should someone else have also checked this? Were there not good requirements? I'd use this as both a place to explain how they made a mistake but also as a learning tool, so make it verbose.
I'll come back to this when there's a bit more here.